engine/src/main/java/org/hibernate/search/query/engine/impl/AbstractHSQuery.java

```diff
@@ -42,9 +43,9 @@
 import org.hibernate.search.spatial.DistanceSortField;
 import org.hibernate.search.spi.CustomTypeMetadata;
 import org.hibernate.search.spi.IndexedTypeIdentifier;
+import org.hibernate.search.spi.IndexedTypeMap;
```

Comment:
just curious: seems something special happened on this case?

Reply:
I probably organized the imports on this and didn't notice